runner.js

```diff
@@ -29,12 +33,17 @@ async function data_routine() {
   const Token0USDRate = await getPoolexchangeRate(
     process.env.TOKEN0_USDC_POOL_ADDRESS
   );
+  console.log("Token0USDRate", Token0USDRate);
```

Comment:
In all following logs, take into acount that we init with possibly 1000s of positions and anyway the system is always up. Do we need those log?

I suggest logging when adding a position for the first time and when removing it

Reply:
removed all console log, this all outdated :)